### PR TITLE
Build game state

### PR DESCRIPTION
**Pull request by RocClark**
still need to give a margin and a boarder for phone view

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,22 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Page Title</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <!-- Latest compiled and minified CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
-    <!-- Optional theme -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
```
Comment (kalobalu) [inline index.html]: spacing

File: index.html [hunk 2]
```diff
@@ -5,22 +5,42 @@
...
+                One of three columns
+              </div>
+              <div class="col-sm">
+                One of three columns
+              </div>
+              <div class="col-sm">
+                One of three columns
+              </div>
+            </div>
+          </div>
```
Comment (kalobalu) [inline index.html]: can remove

File: main.css [hunk 3]
```diff
@@ -6,10 +6,19 @@ a {
 color: green;
 }
 
+.game-area{
+  padding-top: 3rem;
+}
+
+.gameboard{
+  height: 100vh;
```
Comment (kalobalu) [inline main.css:14]: what does vh and vw mean?

File: tictaktoe.html [hunk 4]
```diff
@@ -6,48 +6,71 @@
...
+        <div class="row score-area">
+            <div id="mainPlayer" class="col player">
+                <div class=" col in-game d-none">
+                    <p><span class="player-name"></p>
+                    <p>click count <span>0</span></p>
+                </div>
+            </div>
+            <div class="col pre-game">
+                <form id="form-player">
+                    First Name: <input class="f-name" type="text" name="fname>"><br>
```
Comment (kalobalu) [inline tictaktoe.html]: want to avoid <br> tags

File: tictaktoe.html [hunk 5]
```diff
@@ -6,48 +6,71 @@
...
+                </div>
+            </div>
+           
+        </div>
+        <div class="row game-area">
+            <div class="col gamePlace box">
                 box-A1
             </div>
-            <div class="col gamePlace" id ="box-A2">
+            <div class="col gamePlace box">
```
Comment (kalobalu) [inline tictaktoe.html]: we can move the id   into the class

File: tictaktoe.html [hunk 6]
```diff
@@ -64,8 +89,14 @@
     <p><a href="./index.html">home</a><p>
 
     <!-- Latest compiled and minified JavaScript -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+<!-- jQuery library -->
```
Comment (kalobalu) [inline tictaktoe.html]: spacing

File: tictaktoe.html [hunk 7]
```diff
@@ -64,8 +89,14 @@
...
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+<!-- Popper JS -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+
+<!-- Latest compiled JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
```
Comment (kalobalu) [inline tictaktoe.html]: spacing

**Review by kalobalu — CHANGES_REQUESTED** [review]

File: tictaktoe.html (hunk 4 above)
Comment (RocClark) [reply tictaktoe.html]: don't know what you mean here

File: main.css (hunk 3 above)
Comment (RocClark) [reply main.css:14]: vh = view height / vw = view width 
new discovery from the css-max = video

File: index.html (hunk 2 above)
Comment (RocClark) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (RocClark) [reply index.html]: done

File: tictaktoe.html (hunk 6 above)
Comment (RocClark) [reply tictaktoe.html]: done

File: tictaktoe.html (hunk 7 above)
Comment (RocClark) [reply tictaktoe.html]: done

**Review by kalobalu — APPROVED** [review]

**Pull request merged**